common/model-views.cpp

```diff
@@ -3167,6 +3208,163 @@ namespace rs2
         }
     }
 
+    void viewer_model::draw_color_ruler(const mouse_info& mouse,
+                                        const stream_model& s_model,
+                                        const rect& stream_rect,
+                                        std::vector<rgb_per_distance> rgb_per_distance_vec,
+                                        double ruler_length, const std::string& ruler_units)
+    {
+        assert(ruler_length > 0.0);
```

Comment:
From my opinion, if I concentrate all the hard-coded values together in one place it will be more difficult to understand/maintenance because it will lose the context and the code flow.